vnext/ReactUWP/Modules/Animated/NativeAnimatedNodeManager.cpp

```diff
@@ -0,0 +1,300 @@
+        }
+        case AnimatedNodeType::Diffclamp:
+        {
+          m_valueNodes.emplace(tag, std::make_shared<DiffClampAnimatedNode>(DiffClampAnimatedNode(tag, config, manager)));
+          break;
+        }
+        case AnimatedNodeType::Transform:
+        {
+          m_transformNodes.emplace(tag, std::make_shared<TransformAnimatedNode>(TransformAnimatedNode(tag, config, manager)));
+          break;
+        }
+        case AnimatedNodeType::Tracking:
+        {
+          break;
+        }
+        default:
+        {
+          assert(false);
+          break;
+        }
+      }
+    }
+
+    void NativeAnimatedNodeManager::ConnectAnimatedNodeToView(int64_t propsNodeTag, int64_t viewTag)
+    {
+      auto node = m_propsNodes.at(propsNodeTag);
+      node->ConnectToView(viewTag);
+    }
+
+    void NativeAnimatedNodeManager::DisconnectAnimatedNodeToView(int64_t propsNodeTag, int64_t viewTag)
+    {
+      auto node = m_propsNodes.at(propsNodeTag);
+      node->DisconnectFromView(viewTag);
+    }
+
+    void NativeAnimatedNodeManager::ConnectAnimatedNode(int64_t parentNodeTag, int64_t childNodeTag)
+    {
+      if (auto parentNode = GetAnimatedNode(parentNodeTag))
+      {
+        if (auto childNode = GetAnimatedNode(childNodeTag))
+        {
+          parentNode->AddChild(childNode);
+        }
+      }
+    }
+
+    void NativeAnimatedNodeManager::DisconnectAnimatedNode(int64_t parentNodeTag, int64_t childNodeTag)
+    {
+      if (auto parentNode = GetAnimatedNode(parentNodeTag))
+      {
+        parentNode->RemoveChild(childNodeTag);
+      }
+    }
+
+    void NativeAnimatedNodeManager::StopAnimation(int64_t animationId)
+    {
+      if (m_activeAnimations.count(animationId))
+      {
+        if (auto animation = m_activeAnimations.at(animationId))
+        {
+          //Do some callback on animation ...
+
+          m_activeAnimations.erase(animationId);
+        }
+      }
+    }
+
+    void NativeAnimatedNodeManager::StartAnimatingNode(int64_t animationId, int64_t animatedNodeTag, const folly::dynamic& animationConfig, const Callback& endCallback)
+    {
+      auto animation = [animationId, animatedNode = m_valueNodes.at(animatedNodeTag), endCallback, animationConfig]() {
+        //return std::make_shared<AnimatedNode>(ValueAnimatedNode(tag, config));
+        //return winrt::make_self<ValueAnimatedNode>(tag, config);
+        switch (AnimationTypeFromString(animationConfig.find("type").dereference().second.getString()))
+        {
+        case AnimationType::Decay:
+          return static_cast<std::shared_ptr<AnimationDriver>>(std::make_shared<DecayAnimationDriver>(DecayAnimationDriver(animationId, animatedNode, endCallback, animationConfig)));
+        case AnimationType::Frames:
+          return static_cast<std::shared_ptr<AnimationDriver>>(std::make_shared<FrameAnimationDriver>(FrameAnimationDriver(animationId, animatedNode, endCallback, animationConfig)));
+        //case AnimationType::Spring:
```

Comment:
>case AnimationType::Spring: [](start = 10, length = 27)

Another TODO that can be fixed later.

Question:  What happens if you specify Spring?  Do we crash?  If so can we make it no-op for now?